types/index.d.ts

```diff
@@ -0,0 +1,5 @@
+import {Plugin} from 'unified'
```

Comment:
Since this depends on unified, I believe it depends on TypeScript 3.4 and above. :thinking: 

```suggestion
// TypeScript Version: 3.4

import {Plugin} from 'unified'
```